app/ui-utils/client/lib/modal.html

```diff
@@ -53,7 +53,7 @@ <h1 class="rc-modal__title">
 					</section>
 					{{# if showFooter }}
 						<footer class="rc-modal__footer {{#unless showFooter}}rc-modal__footer--empty{{/unless}}">
-							<input class="rc-button rc-button--nude js-close {{#unless showCancelButton}}rc-button--invisible{{/unless}}" style="{{#if cancelButtonColor}}color: {{cancelButtonColor}}{{/if}}" type="submit" data-button="cancel" value="{{cancelButtonText}}">
+							<input class="rc-button rc-button--nude js-close {{#unless showCancelButton}}rc-button--invisible{{/unless}}" style="background-color: #bdbebf;{{#if cancelButtonColor}}color: {{cancelButtonColor}}{{/if}}" type="submit" data-button="cancel" value="{{cancelButtonText}}">
```

Comment:
I think it's better to use the `rc-button--secondary` class since it adds this exact color as background-color while allows customization by administrator.

Reply:
Done  : )